src/graph/validator/MutateValidator.h

```diff
@@ -38,6 +38,20 @@ class InsertVerticesValidator final : public Validator {
   std::vector<storage::cpp2::NewVertex> vertices_;
 };
 
+class InsertVerticesOnlyValidator final : public Validator {
```

Comment:
Why don't reuse InsertVerticesValidator?

Reply:
The syntax of `InsertVerticesOnly` and the syntax of `InsertVertices` are quite different in the parser process. This causes the data structure contained in the two Sentences to be different. If both Sentences use `InsertVerticesValidator`, two completely different `toPlan` logics are also required. So simply use a new `InsertVerticesOnlyValidator`. This also ensures that the impact on `InsertVertices` is minimal, and other bugs will not be introduced during the modification process.

Reply:
https://github.com/vesoft-inc/nebula/blob/df3781cb1b530d93a6aaaff0f032cf626884f2a5/src/parser/parser.yy#L2839

I think just need add a new reduce rule like `vid { $$ = new VertexRowIten($1, new ValueList())}` in `vertex_row_item` rule.

Reply:
But, `InsertVertices` can not use vertexID list as "1,2,3".It needs "1:(),2:(),3:()" instead. So if add a new reduce rule in `vertex_row_item`, the parse process of `insertVertices` will be very strange.